src/models/log.ts

```diff
@@ -1,38 +1,82 @@
+import { Department } from "../constansts/department"
 import { db } from "../database/db"
 import { CreateLog, EditLog, Log } from "../types/log"
+import { SqlResultObject } from "../types/types"
+import { CommentModel } from "./comment"
 
 const create = async (createLog: CreateLog): Promise<boolean> => {
     const query = "INSERT INTO `LOG` SET ?"
     const date = new Date()
-    const result = await db.query(query, [{ ...createLog, createdDate: date }])
+    const result: SqlResultObject = await db.query(query, [{ ...createLog, createdDate: date }])
     return (result.insertId) ? true : false
 }
 
-const find = async (param: { id: number } | { employeeId: number }): Promise<Log[]> => {
-    const query = "SELECT * FROM `LOG` WHERE ?"
-    const result: Log[] = await db.query(query, [param])
-    return result
+const find = async (param?: { key: 'id' | 'employeeId', value: number }): Promise<Log[]> => {
```

Comment:
we are finding employee by employee id and employee id is unique so, how can results will be multiple